README.md

```diff
@@ -68,28 +88,3 @@ miner-configs:
   f05678:
     max-concurrent-retrievals: 10
 ```
-
-## Help
-```console
-$ autoretrieve --help
```

Comment:
Could we keep at least keep just this section for now if we're not replacing it with a run guide of some kind?

Reply:
Opened an issue for adding a run guide #161